test/helpers/content-claims.js

```diff
@@ -122,41 +122,51 @@ export const generateClaims = async (signer, dataCid, carCid, carStream, indexCi
  * @param {import('cardex/api').CARLink} carCid
  * @param {ReadableStream<Uint8Array>} carStream CAR file data
  */
-export const generateLocationClaims = async (signer, carCid, carStream) => {
+export const generateBlockLocationClaims = async (signer, carCid, carStream) => {
   /** @type {Claims} */
   const claims = new LinkMap()
 
   await carStream
     .pipeThrough(new CARReaderStream())
     .pipeTo(new WritableStream({
       async write ({ cid, blockOffset, blockLength }) {
-        const invocation = Assert.location.invoke({
-          issuer: signer,
-          audience: signer,
-          with: signer.did(),
-          nb: {
-            content: cid,
-            location: [
-              /** @type {import('@ucanto/interface').URI<'http:'>} */
-              (`http://localhost/${carCid}/${carCid}.car`)
-            ],
-            range: { offset: blockOffset, length: blockLength }
-          }
-        })
-
         const blocks = claims.get(cid) ?? []
-        // @ts-expect-error
-        blocks.push(await encode(invocation))
+        const location = new URL(`https://w3s.link/ipfs/${carCid}?format=raw`)
```

Comment:
I think we should use raw CIDs here as opposed to CAR CIDs with `format=raw` should not we ?